bridge/client/app/app.routing.ts

```diff
@@ -32,7 +32,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })],
```

Comment:
please add a docstring telling what `relativeLinkResolution` is doing

Reply:
Migration added this automatically.
Angular 11 introduced a bugfix for the usage of relative paths. https://angular.io/api/router/ExtraOptions#relativeLinkResolution
We are not using that anywhere - so we could get rid of it.

Reply:
This will be removed, as it is not needed in Bridge